doc/user/content/self-hosted/configuration.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Materialize Operator Configuration"
+description: ""
+---
+
+You can configure the Materialize operator chart. For example:
+
+- **RBAC**
```

Comment:
Below where I have the ## Parameters section, I could section off the parameters ... so that we don't need to "For example" here.  

That is, in the parameters section, I could create subsections

```
## Parameters

### Network policies

### Observability

### RBAC

```